src/modules/roc_fec/reader.cpp

```diff
@@ -385,39 +333,17 @@ void Reader::update_source_block_() {
         // should not happen: we have handled preceding and following blocks above
         roc_panic_if_not(fec.source_block_number == cur_sbn_);
 
-        if (!validate_incoming_source_packet_(pp)) {
+        if (!process_source_packet_(pp)) {
```

Comment:
s/process_source_packet_/handle_source_packet_ ?
The same for the repair packets.

Reply:
What's the difference?